modules/ROOT/pages/troubleshoot-api-provisioning-issues.adoc

```diff
@@ -4,8 +4,13 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: service mesh, microservices, API provisioning issues, troubleshooting
 
-After you provision an adapter, you must create an API, to which you then bind the service. You encounter errors when you provision an API if you provide invalid credentials.
 
+After you provision an adapter, you must create an API to bind the service to, which might require you to troubleshoot the following errors: 
+
+* <<client-secret-incorrectly-configured,Client secret incorrectly configured>>
+* <<provision-call-failed,API provisioning failure>>
+
+[[client-secret-incorrectly-configured]]
 == Client Secret is Incorrectly Configured
```

Comment:
Verbs are always init cap it titles and headings:
```suggestion
== Client Secret Is Incorrectly Configured
```
OR
```suggestion
== Incorrectly Configured Client Secret
```
if you want to turn error message text into a parallel topic heading

Reply:
Done! Thanks!